src/options/options.js

```diff
@@ -190,6 +263,41 @@ const restoreOptions = async () => {
     }
   }
 
+  const { optionsPageHasBeenShown } = await messenger.storage.local.get({
+    optionsPageHasBeenShown: false,
+  });
+
+  if (!optionsPageHasBeenShown) {
```

Comment:
That doesn't work because the `background.js` page has already set the value to `true` here.

Actually, I think I would prefere to remove that option to close the "native script" section, if you don't mind. I may later on code something to automatically save the options without requiring a "save" button.

Reply:
Oh, well, too bad!

Would if work, if those two calls in `background.js` are swapped, so that the option page is opened before the `optionsPageHasBeenShown` set to `true`?
```javascript
    await messenger.storage.local.set({
      optionsPageHasBeenShown: true,
    });
    await browser.runtime.openOptionsPage();
```

Reply:
I would set `optionsPageHasBeenShown` to `true` _in the options page itself_ then, if you want to keep your closable section.

Reply:
That section about installing the native script always distracts me, when I go into the settings page …

I moved it to the options page itself.